src/content/apply/design.md

```diff
@@ -3,9 +3,9 @@ qualities:
   - Has a passion for creating brand identity
   - Is experienced with graphic design, including tools like the Adobe suite, Figma, etc.
   - Wants to create applications that both look great and have an intuitive user experience
-formLink: "https://docs.google.com/forms/d/e/1FAIpQLSci8o8K8I4szGhfdqiJ5sAfB9NHgdv6JGixMNy7lV9aW0EjqA/viewform?usp=sf_link"
+formLink: "http://eepurl.com/gyY5lz"
 role: designer
-closeDate: August 7th (rolling)
+closeDate: null
```

Comment:
Instead of the createTypes stuff, could we just make closeDate some other falsey value, like 0?

Reply:
Yeah TBH I don't even fully understand why it works... I think it just infers the types that you don't specify